model/project_ref.go

```diff
@@ -339,15 +339,15 @@ const (
 type ProjectPageSection string
 
 const (
```

Comment:
Can we drop a comment here that these need to stay consistent with the graphql enums?